web_client/src/components/Navbar.vue

```diff
@@ -72,10 +72,9 @@ export default defineComponent({
         <template #activator="{ on }">
           <span v-on="on">MIQA</span>
         </template>
-        <span>{{
-          MIQAConfig.version && MIQAConfig.version.length > 0
-            ? MIQAConfig.version
-            : "Demo Instance"
+        <span>{{ MIQAConfig.version.length > 0
+          ? MIQAConfig.version
+          : "Demo Instance"
```

Comment:
Yep, this is what I meant, though I would then just change this to:

```
<span>{{ MIQAConfig.version || "Demo Instance" }}</span>

Reply:
Changed in [536e8b1](https://github.com/OpenImaging/miqa/pull/630/commits/536e8b1799d940291940589960350e4befa96ed6)